apps/demos/Demos/Chat/Overview/jQuery/index.js

```diff
@@ -0,0 +1,89 @@
+$(async () => {
```

Comment:
Why this function should be async?

Reply:
b17c620427bf8b4a5a3c88d77addd374fa962cf2